components/wpt-results.html

```diff
@@ -224,9 +230,19 @@
           this._refreshDisplayedNodes()
         }
 
-        const testFileResults = await Promise.all(this.testRuns.map(testRun => {
-          return this._fetchResults(testRun.results_url)
-        }))
+        const testRuns = await Promise.all(
+          this.testRunSpecs.map(async testRun => {
+            const pieces = testRun.split('@')
+            const browser = pieces[0]
+            const sha = pieces.length > 1 ? pieces[1] : 'latest'
+            const url = `/api/run?browser=${browser}&sha=${sha}`
+            const response = await window.fetch(url)
+            return response.json()
+          }))
+
+        const testFileResults = await Promise.all(
+          testRuns.map(async json => { return this._fetchResults(json.results_url) })
```

Comment:
The `return` and curly braces are still unnecessary here (unless the linter is making a fuss)

Reply:
Done.